bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TableValueImpl.java

```diff
@@ -46,6 +46,7 @@
 import io.ballerina.runtime.internal.types.BTupleType;
 import io.ballerina.runtime.internal.types.BTypeReferenceType;
 import io.ballerina.runtime.internal.types.BUnionType;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
Shall we avoid importing `org.apache.commons.lang3` and use another way to create a pair such as creating a private static class. We decided to remove those external library usages to reduce the jar size.